### DISCUSSION TITLE
Add audio to replays

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#5180] [Ideas]
It would be great if users could talk over their recordings in a Loom-style way so that it is easier to follow what they're doing.

Just adding text-comments afterwards is inneficient.

**Comment by jonbell-lot23** [comment]
Ah, the perennial feedback! For a long time our POV was that we don't want to re-build Loom. But this is definitely a common request.

**Comment by TJPain** [comment]
I've just started using Replay in my new job - it's great but I do really miss the voice aspect of Loom, which we used at my old job. 

You just get so much more context and information from the person that made the video with them talking you through things than you do from a couple of lines of text as a note. In addition, I think that people are more likely to report the bugs with a video as you can make the videos much more easily / quickly.

**Reply by jasonLaster** [reply]
Agreed. We use Loom a lot on the team as well and the audio component is incredibly helpful.

**Reply by jonbell-lot23** [reply]
Also agree

**Comment by ceceliacreates** [comment]
Received user feedback requesting this on 6/17.

>  do you guys offer audio recording so that I can verbally narrate what I'm doing?
